auth/auth.go

```diff
@@ -5,6 +5,7 @@ import (
 	"os"
 
 	"github.com/google/go-github/v32/github"
+	"github.com/gruntwork-io/git-xargs/ratelimiting"
```

Comment:
Total NIT, but consider calling the package `ratelimit` because it's the shortest thing that still makes sense.